docs/admin-guide.md

```diff
@@ -0,0 +1,8 @@
+# Admin guide
+
+- Installation
+    - [Manual](#manual-installation)
+
+## Manual Installation
+
+If your server doesn't have access to the internet, you can download the [latest plugin binary release](https://github.com/mattermost/mattermost-plugin-splunk/releases) and upload it to your server via **System Console > Plugin Management**. The releases on this page are the same used by the Marketplace. To learn more about how to upload a plugin, see the [Mattermost Developer documentation](https://developers.mattermost.com/integrate/plugins/using-and-managing-plugins/).
```

Comment:
I think we can just keep this section in the README and remove this file. We can just call the section "Installation". Then change the content to:

```suggestion

# Installation

You can download the [latest plugin binary release](https://github.com/mattermost/mattermost-plugin-splunk/releases) and upload it to your server via **System Console > Plugin Management**.
```